unstoppable-ios-app/domains-manager-ios/SupportingFiles/Constants.swift

```diff
@@ -227,3 +227,19 @@ struct Utilities {
         }
     }
 }
+
+let cartPriceFormatter: NumberFormatter = {
+    let formatter = NumberFormatter()
+    formatter.numberStyle = .currency
+    formatter.locale = Locale(identifier: "es_CL")
```

Comment:
Spanish Chile?
Also this constant deserves to be extracted at some global level